executors/dart_web/bin/collator.dart

```diff
@@ -22,10 +22,20 @@ String testCollationShort(String jsonEncoded) {
     } else {
       coll = Intl();
     }
-    var d1 = json['string1'];
-    var d2 = json['string2'];
+    var d1 = json['s1'];
```

Comment:
This should work:
```
var d1 = json['s1'];
var d2 = json['s2'];

var collationOptions = CollationOptions(
      ignorePunctuation: json['ignorePunctuation'] ?? false,
);
```

Reply:
(If the json contains a string and not a bool, it should rather be
```
var d1 = json['s1'];
var d2 = json['s2'];

var collationOptions = CollationOptions(
  ignorePunctuation: bool.tryParse(json['ignorePunctuation']) ?? false,
);
```
)

Reply:
Thanks. I'm integrating these, changing "var" to "final".

Reply:
This now appears to work with dart_web. Thank you!